src/components/prediction/DataElement.tsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from "prop-types";
 import i18n from "@dhis2/d2-i18n";
 import { useDataQuery } from "@dhis2/app-runtime";
 import { SingleSelectField, SingleSelectOption } from "@dhis2/ui";
+import React from "react";
```

Comment:
Better to include with useEffect on line 1:

`import React, { useEffect } from "react";`